scapy/contrib/gtp_v2.py

```diff
@@ -464,7 +464,7 @@ class PCO_Secondary_NBNS(PCO_Option):
     0x000c: 'P-CSCF IPv4 Address Request',
     0x0010: 'IPv4 Link MTU Request',
     0x8021: 'IPCP',
-    0xc023: 'Password Authentification Protocol',
```

Comment:
I looked up that one, and it seems correct: Authentification is French, Authentication seems to be correct